config.xml

```diff
@@ -108,8 +108,6 @@
     </platform>
     <preference name="SplashScreen" value="screen" />
     <preference name="ShowSplashScreenSpinner" value="false" />
-    <preference name="StatusBarOverlaysWebView" value="false" />
-    <preference name="StatusBarBackgroundColor" value="#141414" />
```

Comment:
Doesn't this need to be replaced in capacitor status-bar? How are these colors set in the new plugin?

Reply:
The equivalent initial configuration has been added at the initial commit of this Branch and more specifically at capacitor.config.ts folder lines 16,17